packages/design-tokens/package.json

```
@@ -10,7 +10,12 @@
   "bugs": {
     "url": "https://github.com/cultureamp/kaizen-design-system/issues"
   },
+  "bin": {
+    "codemod": "./bin/codemod.sh"
```

Since these become rather global, if we can get this working we should give it a more unique name to prevent clashing
```suggestion
    "kaizen-codemod": "./bin/codemod.sh"
```